src/main/java/de/ait/secondlife/services/OfferServiceImpl.java

```diff
@@ -142,13 +149,14 @@ public OfferResponseDto updateOffer(OfferUpdateDto dto) throws CredentialExcepti
         } else {
             draftOffer(offer);
         }
-
+        imageService.connectTempImgsToEntity(dto.getBaseNameOfImgs(), dto.getId());
```

Comment:
problems arise here if the request again includes the base names of the images that were already attached to the entity. It seems like this is acceptable client behavior and they don't have to guess that the image attaching only works once. the code should not break and should be prepared to re-handle basenames

Reply:
fixed